src/gfnff/gfnff_eg.f90

```diff
@@ -4140,10 +4140,8 @@ subroutine es_grad_sigma(mol, topo, nlist, rTrans, gTrans, xtmp, cf, &
    real(wp), allocatable :: dXvecdr(:,:,:)
    real(wp), allocatable :: amatdr(:, :, :)
    real(wp), allocatable :: amatdL(:, :, :)
-   integer :: m
    real(wp), allocatable :: atrace(:, :)
-   m = mol%n+topo%nfrag
-   allocate(amatdr(3,mol%n,m), amatdL(3,3,m), source = 0.0_wp)
+   allocate(amatdr(3,mol%n,mol%n), amatdL(3,3,mol%n), source = 0.0_wp)
    allocate(atrace(3, mol%n))
```

Comment:
```suggestion
   allocate(atrace(3, mol%n), source = 0.0_wp)
```

Reply:
Just for consistency. This array seems to be properly zero'd in the `get_damat` subroutine.

Reply:
Applied.